devimint/src/lib.rs

```diff
@@ -141,7 +141,7 @@ impl Gatewayd {
 
     pub async fn connect_fed(&self, fed: &Federation) -> Result<()> {
         let invite_code = fed.invite_code()?;
-        poll_max_retries("gateway connect-fed", 10, || async {
+        poll_max_retries("gateway connect-fed", 60, || async {
```

Comment:
Is this related to #2971?

Reply:
Yes. Removing it as it was moved to another PR.